pkg/routing/selector/utils.go

```diff
@@ -62,7 +62,12 @@ func LimitsReached(limitConfig config.LimitConfig, nodeStats *livekit.NodeStats)
 	if limitConfig.NumTracks > 0 && limitConfig.NumTracks <= nodeStats.NumTracksIn+nodeStats.NumTracksOut {
 		return true
 	}
-	if limitConfig.BytesPerSec > 0 && limitConfig.BytesPerSec <= nodeStats.BytesInPerSec+nodeStats.BytesOutPerSec {
+
+	rate := &livekit.NodeStatsRate{}
+	if len(nodeStats.Rates) > 0 {
+		rate = nodeStats.Rates[0]
```

Comment:
using the first configured rate interval for these cases. Default is only one rate interval which is set to 10 seconds. If users reconfigure to use more, they can either ensure that the first one is the relevant one for selector or augment this to either use a different interval or some combination  of rates from different intervals etc.